velox/exec/Operator.h

```diff
@@ -416,7 +428,8 @@ class Operator : public BaseRuntimeStatWriter {
   static std::unique_ptr<Operator> fromPlanNode(
       DriverCtx* FOLLY_NONNULL ctx,
       int32_t id,
-      const core::PlanNodePtr& planNode);
+      const core::PlanNodePtr& planNode,
+      std::shared_ptr<ExchangeClient> exchangeClient);
```

Comment:
Maybe clarify that exchangeClient is not-null only when planNode->requiresExchangeClient() is true.